public/src/components/Spinner/Spinner.js

```diff
@@ -0,0 +1,29 @@
+import React, {useState} from 'react'
+import {useSelector} from "react-redux";
+import Loader from "react-loader-spinner";
+
+
+function Spinner(props) {
```

Comment:
You branched this code from you spinner branch not the develop branch you will need to remove the spinner changes

Reply:
Im probably not going to do that. Im afraid I will mess things up. This will have to have both the contact and spinner branch changes in it until I understand Github and branching better and how to keep them seperate.

Reply:
If you do this it very likely will result in merge conflicts which will only make things more difficult.. 

Just remove all spinner files and use of spinner components from this branch